java/distance-composite-symbol/README.md

```diff
@@ -1,36 +1,31 @@
-# Distance Composite Symbol
-Create a graphic using a distance composite scene symbol. Distance composite scene symbols can render different symbols depending on the distance between the camera and the graphic.
+# Distance composite symbol
 
-![Distance Composite Symbol App](distance-composite-symbol.png)
+Change a graphic's symbol based on the camera's proximity to it.
+
+![Image of distance composite symbol](distance-composite-symbol.png)
+
+## Use case
+
+When showing dense datasets, it is beneficial to reduce the detail of individual points when zooming out to avoid visual clutter and to avoid data points overlapping and obscuring each other.
 
 ## How to use the sample
-Zoom in and out to see the symbol of the graphic change.
+
+The sample starts looking at a plane. Zoom out from the plane to see it turn into a cone. Keeping zooming out and it will turn into a point.
 
 ## How it works
-To create and display a `DistanceCompositeSceneSymbol`:
 
-1. Create a `GraphicsOverlay` and add it to the `SceneView`.
-1. Create symbols for each `Range` of the composite symbol.
-1. Create a `DistanceCompositeSceneSymbol`.
-1. Add a range for each symbol to `DistanceCompositeSceneSymbol.getRangeCollection().add(new Range(symbol, min distance, max distance))`.
-  * symbol to be used within the min/max range that is given
-  * min and max distance that the symbol will be display at from the `Camera`
-1. Create a `Graphic` with the symbol: `Graphic(Point, DistanceCompositeSceneSymbol)`
-1. Add the `Graphic` to the `GraphicsOverlay`.
+1. Create a `GraphicsOverlay` object and add it to a `SceneView` object.
+2. Create a `DistanceCompositeSceneSymbol` object.
+3. Create `DistanceCompositeSceneSymbol.Range` objects specifying a `Symbol` and the min and max distance within which the symbol should be visible.
+4. Add the ranges to the range collection of the distance composite scene symbol.
+5. Create a `Graphic` object with the distance composite scene symbol at a location and add it to the graphics overlay.
 
 ## Relevant API
-* ArcGISScene
-* ArcGISTiledElevationSource
-* Camera
+
 * DistanceCompositeSceneSymbol
-* DistanceCompositeSceneSymbol.Range
-* Graphic
-* GraphicsOverlay
-* ModelSceneSymbol 
 * Range
```

Comment:
since both of these belong to `DistanceCompositeSceneSymbol` and aren't individual classes in their own right, we should prefix them with their class 
```suggestion
* DistanceCompositeSceneSymbol.Range
* DistanceCompositeSceneSymbol.getRangeCollection
```